src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/configure-details.jelly

```diff
@@ -26,7 +26,9 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project">
     <st:include page="configure-entries" optional="true"/>
-    <p:config-trigger/>
+    <f:descriptorList title="${%triggers(it.computation.displayName)}"
+                      descriptors="${it.triggerDescriptors}"
+                      instances="${it.triggers}"/>
```

Comment:
I think this only actually works by accident in `config-trigger.jelly`. Better style is to use the `field` attribute.

Either way, you need a `configRoundtrip` test verifying that the databinding code is correct.